charts/datagovuk/templates/_publish.tpl

```diff
@@ -1,22 +1,24 @@
 {{- define "publish.environment-variables" -}}
 {{- $environment := eq .Values.environment "test" | ternary "development" .Values.environment -}}
+{{- with .Values.publish.config }}
 - name: CKAN_URL
-  value: http://{{ .Values.publish.config.ckanReleaseName }}-ckan:5000
+  value: http://{{ .ckanReleaseName }}-ckan:5000
 - name: DATABASE_URL
   valueFrom:
     secretKeyRef:
-      name: {{ .Values.publish.config.sqlalchemyUrlSecretKeyRef.name }}
-      key: {{ .Values.publish.config.sqlalchemyUrlSecretKeyRef.key }}
+      name: {{ .sqlalchemyUrlSecretKeyRef.name }}
+      key: {{ .sqlalchemyUrlSecretKeyRef.key }}
 - name: ES_HOST
   value: http://{{ $.Release.Name }}-opensearch:9200
 - name: RAILS_ENV
   value: {{ $environment }}
 - name: REDIS_HOST
-  value: {{ .Values.publish.config.ckanReleaseName }}-redis
+  value: redis://{{ .redis.host | default "ckan-redis" }}/{{ .redis.db_number | default "1" }}
```

Comment:
```suggestion
  value: redis://{{ .redis.host | default "ckan-redis" }}/{{ .redis.dbNumber | default "1" }}
```

Reply:
Yeah (apologies if I missed this in an earlier review!) (and ofc we should just have a lint rule for this sort of trivial stuff but apparently chart-testing doesn't catch it for some reason). Thanks!